### PR TITLE
v5.2.0

### LINKED ISSUE #492: improv(ingress): update to Kube 1.19
```
Warning: extensions/v1beta1 Ingress is deprecated in v1.14+, unavailable in v1.22+; use networking.k8s.io/v1 Ingress
```

### PR DESCRIPTION
**Pull request by douglasduteil**
---

- [ ] add doc
- [ ] add remote create db per branch
- [x] add remote remove db per branch
- [x] update to networking.k8s.io/v1 ingress api

Closes #492

### CODE REVIEW

**Comment by douglasduteil** [comment]
Demo https://gitlab.factory.social.gouv.fr/SocialGouv/cdtn-admin/-/jobs/1375118

image

File: src/utils/createIngress.ts [hunk 1]
```diff
@@ -16,13 +17,21 @@ export interface IngressConfig {
   annotations?: Record<string, unknown>;
 }
 
-const getHostService = ({ serviceName = "app", servicePort = 3000 }) => ({
+const getHostService = ({
+  serviceName = "www",
+  servicePort = 80,
```
Comment (revolunet) [inline src/utils/createIngress.ts:22]: isnt this breaking change ?

File: templates/autodevops/components/jobs/dropdb/index.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { dropDbJob } from "@socialgouv/kosko-charts/components/azure-pg/drop-db.job";
+import { getDevDatabaseParameters } from "@socialgouv/kosko-charts/components/azure-pg/params";
+import { ok } from "assert";
+
+const suffix = process.env.ENVIRONMENT_SLUG ?? process.env.CI_ENVIRONMENT_SLUG;
```
Comment (revolunet) [inline templates/autodevops/components/jobs/dropdb/index.ts:5]: we should centralize that somwhere

File: templates/autodevops/kosko.toml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+components = ["!jobs", "*"]
+require = ["ts-node/register"]
```
Comment (revolunet) [inline templates/autodevops/kosko.toml:2]:
```suggestion
require = ["ts-node/register"]

[environments.prod]
components = ["!_*", "!jobs", "*"]

```

File: templates/autodevops/components/jobs/dropdb/index.ts (hunk 2 above)
Comment (revolunet) [reply templates/autodevops/components/jobs/dropdb/index.ts:5]: `utils/getPgParams` ?

**Comment by revolunet** [comment]
i think we should also resolve the db-per-branch issue here, i tried here : https://github.com/SocialGouv/kosko-charts/pull/448/files?file-filters%5B%5D=.ts#diff-18a4858b9e1da85e6a9d3bc46fb0ec246820caf60b52af39255460dc39b20c2f

**Comment by douglasduteil** [comment]
Outdated

**Pull request closed without merging**